site/it/community/contribute/index.md

```diff
@@ -0,0 +1,49 @@
+
+Tra gli esempi di comportamenti che contribuiscono a creare un ambiente positivo troviamo: 
+
+*  Utilizzare un linguaggio gradito e aperto a tutti.
+*   Avere rispetto dei diversi punti di vista e delle esperienze.
+*   Accettare con dignità la critica costruttiva.
+*   Favorire cosa è meglio per la comunità.
+*   Mostrare empatia nei confronti degli altri membri della comunità.
+
+Le decisioni vengono prese in base a meriti tecnici e al consenso. La comunità TensorFlow aspira a trattare ognuno allo stesso modo e a dare valore a tutti i contribuiti.
+ Per ulteriori informazioni sui migliori esercizi all'interno della comunità TensorFlow per favore vedi qui: 
+[Codice di condotta](https://github.com/tensorflow/tensorflow/blob/master/CODE_OF_CONDUCT.md).
+
+## Fai il tuo primo contributo
+
+Ci sono diversi modi per contribuire a TensorFlow! Puoi contribuire con del codice, apportare dei miglioramenti alla documentazione TensorFlow API, oppure aggiungere i tuoi 
+Jupyter notebooks al repository [tensorflow/examples](http://www.github.com/tensorflow/examples).
+Questa guida fornisce tutto quello di cui hai bisogno per iniziare. I nostri contributi più comuni includono: il *codice*, la *documentazione*, e il *supporto alla comunità*.
+
+- [Scrivere codice](code.md).
+- [Migliorare la documentazione](docs.md).
+- Rispondere alle domande su
+  [Stack Overflow](https://stackoverflow.com/questions/tagged/tensorflow).
+- Partecipare alle discussioni sui
+  [forum TensorFlow](https://www.tensorflow.org/community/forums).
+- Contribuire ai [notebook di esempio](http://www.github.com/tensorflow/examples).
+- Investigare i [bug e le issue](https://github.com/tensorflow/tensorflow/issues)
+  su GitHub.
+- Fare la review e commentare le
+  [pull request (https://github.com/tensorflow/tensorflow/pulls) degli altri sviluppatori.
+- [Segnalare un issue](https://github.com/tensorflow/tensorflow/issues/new/choose).
+- Fare un “like” 👍 sugli issue che sono importanti per te.
+- Menzionare TensorFlow nei tuoi blog, documenti, e articoli.
+- Parlare di TensorFlow sui social media.
+- ... anche solo facendo star/fork sui repository che ti piacciono su GitHub!
+
+TensorFlow è stato sviluppato in origine da alcuni ricercatori e ingenieri del gruppo Google
+Brain [Google's AI organization](https://ai.google/). Google ha creato l'open source TensorFlow con la speranza di condividere la tecnologia con una comunità esterna e incoraggia la collaborazione tra i ricercatori e il settore. Da quel momento,
+TensorFlow è cresciuto in un ecosistema di prodotti su delle estese piattaforme. Ma il nostro goal è ancora quello di creare un machine learning accessibile a tutti,
```

Comment:
TensorFlow è cresciuto in un ecosistema di prodotti su delle estese piattaforme. Ma il nostro goal è ancora...
Looking at the English version, in my opinion, this sounds better:
TensorFlow è cresciuto in un fiorente ecosistema di prodotti, su un ampio spettro di piattaforme. Ma il nostro obiettivo è ancora...